src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs

```diff
@@ -234,28 +269,64 @@ private InlineDiffLine<CodeLine>[] CreateDiffOnlyLines(InlineDiffLine<CodeLine>[
             return filteredLines.ToArray();
         }
 
-        private CodeLineModel[] CreateLines(CodeDiagnostic[] diagnostics, InlineDiffLine<CodeLine>[] lines, ReviewCommentsModel comments)
+        private CodeLineModel[] CreateLines(CodeDiagnostic[] diagnostics, InlineDiffLine<CodeLine>[] lines, ReviewCommentsModel comments, bool hideCommentRows = false)
         {
             if (ShowDiffOnly)
             {
                 lines = CreateDiffOnlyLines(lines);
             }
+            List<int> documentedByLines = new List<int>();
+            int lineNumberExcludingDocumentation = 0;
+            int diffSectionId = 0;
 
             return lines.Select(
-                (diffLine, index) => new CodeLineModel(
-                    diffLine.Kind,
-                    diffLine.Line,
-                    diffLine.Kind != DiffLineKind.Removed &&
-                    comments.TryGetThreadForLine(diffLine.Line.ElementId, out var thread) ?
-                        thread :
-                        null,
-
-                    diffLine.Kind != DiffLineKind.Removed ?
-                        diagnostics.Where(d => d.TargetId == diffLine.Line.ElementId).ToArray() :
-                        Array.Empty<CodeDiagnostic>(),
-                    diffLine.Line.LineNumber ?? ++index,
-                    new int[] { }
-                )).ToArray();
+                (diffLine, index) =>
+                {
+                    if (diffLine.Line.IsDocumentation)
+                    {
+                        // documentedByLines must include the index of a line, assuming that documentation lines are counted
+                        documentedByLines.Add(++index);
+                        return new CodeLineModel(
```

Comment:
could we add either parameter name i.e. `<paramenter>:` or add variables for each of the things that are being passed into this constructor? 
This will improve the readability of the code

Reply:
Same for the else block below